Doc/library/os.rst

```diff
@@ -3387,7 +3387,7 @@ written in Python, such as a mail server's external command delivery program.
 
 .. function:: posix_spawn(path, argv, env, *, file_actions=None, \
                           setpgroup=None, resetids=False, setsigmask=(), \
-                          setsigdef=(), scheduler=None)
+                          setsigdef=(), scheduler=None, setsid=True)
```

Comment:
Why is `setsid` `True` by default? It doesn't match the default behavior of `posix_spawn` from the C library.

Reply:
Oops, I missed that. Yeah, it must be False by default.

Reply:
The default is now False, you forgot to update the doc ;-)